src/main.zig

```diff
@@ -315,8 +315,9 @@ const usage_build_generic =
     \\  --subsystem [subsystem]        (windows) /SUBSYSTEM:<subsystem> to the linker\n"
     \\  --stack [size]                 Override default stack size
     \\  --image-base [addr]            Set base address for executable image
+    \\  -syslibroot [dir]              (darwin) prepend a prefix to all search paths
     \\  -framework [name]              (darwin) link against framework
-    \\  -F[dir]                        (darwin) add search path for frameworks
+    \\  -F [dir]                       (darwin) add search path for frameworks
```

Comment:
This change is fine, but just FYI we do actually support both

Reply:
Oh ok. I'm happy to revert!